build.gradle

```diff
@@ -16,7 +16,8 @@ task build {
 // docker & docker run
 docker {
     name "unit8/darts"
-    tags version, "latest"
+    //it will also automatically create image with tag 'latest'
+    tag version, "unit8/darts:${version}"
```

Comment:
will it create three tags? why we need `tag version` ?

Reply:
Let's explicit say latest, rather than assume it is automatic

Reply:
> will it create three tags? why we need `tag version` ?

It will create two tags -  'latest' and 'x.y.z' (where 'x.y.z' is release version)

```version``` is variable that is passed to gradle after resolving current release tag by github action. So if release tag is 0.3.0 ```version``` will be replaced by 0.3.0, and gradle will build docker image with two tags, 'latest' and '0.3.0'. Maybe there is better way to parse current version to gradle. If so, I am open to any further suggestions.

Reply:
so why we need this - "unit8/darts:${version}" ? Is it a formatter?

Reply:
I also think that it is a bit unclear, but I will do my best to explain it. Basically "tags" is deprecated and now it is replaced by "tag". So, we can add any number of following lines:   
`tag x, y`
Now, `x` is internal name, some sort of alias, whereas `y` is `image:tag` that corresponds to image name and demanded tag, so for example we can specify following line: 
`tag unicorn, unit8/darts:0.3.0`  
Purpose of this `key, value` correspondence is to refer to created tags by keys. For example, we can then call:  
`./gradlew dockerPushUnicorn`   
And image with tag **0.3.0** (unit8/darts:0.3.0) will be pushed. It can be useful if we have multiple tags and want to push only subset of them. I really recommend reading [Task section in plugin's README](https://github.com/palantir/gradle-docker#tasks).